app/apptesting/test_suite.go

```diff
@@ -71,7 +71,9 @@ func (s *KeeperTestHelper) SetEpochStartTime() {
 		epoch.StartTime = s.Ctx.BlockTime()
 		epochsKeeper.DeleteEpochInfo(s.Ctx, epoch.Identifier)
 		err := epochsKeeper.AddEpochInfo(s.Ctx, epoch)
-		s.Require().NoError(err)
+		if err != nil {
+			panic(err)
+		}
```

Comment:
Why this change?

Reply:
We can't use any suite methods for the benchmark (until #2554 )